### ISSUE TITLE
Any way to update the databases or add the ability to add custom ones and configure

### ISSUE THREAD

**Issue by Haagimus** [XQS6LB3A/lyrics-module#8]
Curious if the title can be implemented. I find that a lot of the music in my Spotify either pulls the wrong lyrics or returns nothing but invalid search results. What can I do to make this work for me a little better?

**Comment by ddddxxx** [comment]
Yes. You can add new lyrics service by implementing [this protocol](https://github.com/ddddxxx/LyricsKit/blob/673fb7904164f7b6e08a9236f64812dce6fbc917/Sources/LyricsService/Provider/LyricsProvider.swift#L16). Here is an [example](https://github.com/ddddxxx/LyricsKit/blob/master/Sources/LyricsService/Provider/Kugou.swift#L36). If you're not familiar with Swift, just tell me the API, I'll do it. Please note LyricsKit currently only support lyrics with time tag, not plain text.

You can also help improving [the sorting algorithm](https://github.com/ddddxxx/LyricsKit/blob/673fb7904164f7b6e08a9236f64812dce6fbc917/Sources/LyricsService/Lyrics%2BQuality.swift#L25).
